### ISSUE TITLE
Boids/fish

### ISSUE THREAD

**Issue by brunobg** [Corollarium/lajevr#46]
Carregar os modelos e animar os boids

**Comment by hiperlogic** [comment]
Depois dos VAT esse é o que pode trazer algum desafio... compreendi o uso, mas tenho que colocar a mão na massa para ter as dúvidas de fato...
As dúvidas surgirão até o final do dia 21 ou início do dia 22

**Comment by brunobg** [comment]
tem um `pages/boids.vue` que carrega só o boids com um peixe. isso facilita bastante testar sem carregar toda a tralha da cena. Não creio que deve dar desafio aqui, é mais um pouco de pensar e umas tentativas e erros pra acertar os parâmetros (vide linhas 321-326), mas a implementação está feita já.

**Comment by hiperlogic** [comment]
Sim... eu percebi o componente boids e entendi que essa era a intenção... (desenvolvemos esse modelo de raciocínio em conjunto na época do Virtuactor! :) )

**Comment by hiperlogic** [comment]
ficou faltando o "força extra"... que não entendi muito bem como funciona... posicionamento acho que precisa de ajuste... criei um mecanismo pra facilitar adicionar novos... eu acho que facilitou! Mas estou aberto a críticas!
